Consoles/Homebrew.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Homebrew
+description: Guides for homebrewing consoles.
+---
+
+## Guides
+
+### Nintendo
+
+[Wii Guide](https://wii.guide) - Guide made by the RiiConnect team to homebrew the Wii
+
+[Hacks.Guide](https://hacks.guide/) - Guides made by the Hacks.Guide team to homebrew the 3ds, WiiU and PS Vita
+
+[Nintendo Homebrew's Nintendo Switch homebrew guide](https://nh-server.github.io/switch-guide/) - Nintendo Switch guide made by [Nintendo Homebrew](https://discord.gg/C29hYvh). You may consider checking out this [GBATemp thread](https://gbatemp.net/threads/the-ultimate-list-of-mods-to-enter-rcm.502145/) to see other alternatives to enter RCM without soldering, but it is recommended to buy a RCM Jig.
+
+[noahc3's Nintendo Switch homebrew guide](https://switch.homebrew.guide/) - Another Nintendo Switch homebrew guide made by noahc3. This guide uses SDSetup, tool that is not recommended by some members, but other than that this guide is fine.
+
+### Sony
+
+[Hacks.Guide's PS Vita guide](https://vita.hacks.guide/) - Guide to homebrew the PS Vita made by the Hacks.Guide team
+
+### Other
+
+[ConsoleMods](https://consolemods.org/) - Community driven wiki dedicated for modding
+
+
+## Playing pirated games
+This section provides information about playing pirated games listed or written by ourselves, as the listed guides are against piracy.
```

Comment:
you should clarify that its written by you specifically

Reply:
done on 8c5d774